docs/concepts.rst

```diff
@@ -5,9 +5,9 @@ The Product Delivery Model
 -----------------------------
 CumulusCI extends the concepts of the `Org Development Model <https://trailhead.salesforce.com/en/content/learn/modules/org-development-model>`_ and the `Package Development Model <https://trailhead.salesforce.com/en/content/learn/modules/sfdx_dev_model>`_ to form a new model, which we call the Product Delivery Model.
```

Comment:
I suggest covering the product delivery model in the Introduction instead. Then that leaves the remaining content all about structural stuff--projects, orgs and services, tasks, and flows. Those all fit together nicely. I'm commenting on the product deliver model sections only. Leaving the rest of the review to Justin.

Reply:
Still wrapping my head around Product Delivery Model, and where it might fit in with the manual. It _does_ pair well with Introduction . . . but it's also such a big topic that we'd need to be cautious about deterring people from skipping over Key Concepts after getting a deep dive in PDM and its relationship to NPSP.